docs-v2/integrations/all/accelo.mdx

```diff
@@ -1,29 +1,43 @@
 ---
 title: Accelo
 sidebarTitle: Accelo
+provider: accelo
 ---
 
-API configuration: [`accelo`](https://nango.dev/providers.yaml)
+import Overview from "/snippets/overview.mdx";
 
-## Features
+import PreBuiltTooling from "/snippets/generated/accelo/PreBuiltTooling.mdx"
+import PreBuiltUseCases from "/snippets/generated/accelo/PreBuiltUseCases.mdx"
 
-| Features | Status |
-| - | - |
-| [Auth (OAuth)](/guides/getting-started/authorize-an-api-from-your-app) | ✅ |
-| [Sync data](/guides/getting-started/read-from-an-api) | ✅ |
-| [Perform workflows](/guides/getting-started/write-to-an-api) | ✅ |
-| [Proxy requests](/guides/proxy-requests-to-an-api) | ✅ |
-| [Receive webhooks](/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+<Overview />
+<PreBuiltTooling />
+<PreBuiltUseCases />
 
-<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+## Access requirements
+| Pre-Requisites | Status | Comment|
+| - | - | - |
+| Paid dev account | ❓ |  |
+| Paid test account | ❓ |  |
+| Partnership | ❓ | |
+| App review | ❓ |  |
+| Security audit | ❓ | |
 
```

Comment:
I think like the readmes, it will be much easier to split the file in two and always regenerate the top.
Otherwise, if you/we need to change something at scale it will be painful

Reply:
The only section that is generated is the overview and the pre-built use-cases section. Unfortunately the titles of the pages don't always align with providers so it'd take a lot more logic to generate the top of the doc.

Reply:
with what bastien said about modifying everything manually, you can dismiss my comment. 
But ultimately we could still generate a big part of it, would requires more component though